test/core/rules/std_test.py

```diff
@@ -7,6 +8,17 @@
 from sqlfluff.core.rules.std import std_rule_set
 
 
+class RuleTestCase(NamedTuple):
+    """Used like a dataclass by rule tests."""
+
+    rule: str = None
+    desc: str = None
```

Comment:
I forgot I added this already @pwildenhain. The `desc` key exists as a valid key in a rule, but it doesn't do anything.